Core/ContentBlockerRulesUserScript.swift

```diff
@@ -67,7 +67,7 @@ public class ContentBlockerRulesUserScript: NSObject, UserScript {
         guard let blocked = dict[ContentBlockerKey.blocked] as? Bool else { return }
         guard let trackerUrlString = dict[ContentBlockerKey.url] as? String else { return }
         let resourceType = (dict[ContentBlockerKey.resourceType] as? String) ?? "unknown"
-        guard let pageUrlStr = message.webView?.url?.absoluteString ?? dict[ContentBlockerKey.pageUrl] as? String else { return }
+        guard let pageUrlStr = dict[ContentBlockerKey.pageUrl] as? String else { return }
```

Comment:
Do you know of any time we don't get `dict[ContentBlockerKey.pageUrl]`? Removing the `message.webView?.url?.absoluteString` part seems necessary to me to avoid the race condition, but I wonder why it was there in the first place, so assume it was probably to fix something else 🙈

Reply:
@SlayterDev I think that (adding `message.webView?.url?.absoluteString`) is a change you've made - do you remember why?

Reply:
It was related to a bug fix but I'm not sure this ended up affecting anything. It can be removed if needed.